CMakeLists.txt

```diff
@@ -918,11 +921,10 @@ IF (WITH_H2OLOG)
 
     ADD_EXECUTABLE(h2olog ${H2OLOG_SOURCE_FILES})
 
-    SET_PROPERTY(TARGET h2olog
-        PROPERTY CXX_STANDARD 11
-        PROPERTY CXX_STANDARD_REQUIRED ON
-        PROPERTY CXX_EXTENSIONS OFF
-    )
+    SET(CMAKE_CXX_FLAGS "-std=gnu++11 ${CMAKE_CXX_FLAGS}")
+    SET_PROPERTY(TARGET h2olog PROPERTY CXX_STANDARD 11)
+    SET_PROPERTY(TARGET h2olog PROPERTY CXX_STANDARD_REQUIRED ON)
+    SET_PROPERTY(TARGET h2olog PROPERTY CXX_EXTENSIONS ON)
```

Comment:
Maybe we can retain the original code here? I'm a bit concerned about the side-effect of setting `-std=gnu++11` then setting CXX_STANDARD.

I do not think we need to spend too much effort in trying to support people who use older versions of CMake _and_ manually turn on support for h2olog.

Reply:
Ah, it makes sense. Changing CMAKE_CXX_FLAGS globally could affect other targets.

How about f6bfa9b1963f11970cce2d47d2e05dfe7b06ac22? It limits the effect of `std=gnu++11` only to `h2olog`.

Reply:
(talked with kazuho san)

Removed a trick to build h2olog with older versions of cmake. It makes no sense.